src/hooks/useConnection.tsx

```diff
@@ -177,6 +189,7 @@ export function useConnection() {
       await onboardInstance.walletSelect();
       await onboardInstance.walletCheck();
       dispatch({ type: "set onboard", onboard: onboardInstance });
+      dispatch({ type: "set connection status", isConnected: true });
```

Comment:
Does this mean if you get to this line, that the user is considered "connected"? What if the `provider`, `network`, or `onboard` objects don't get initialized properly for example?

Reply:
Yes, and this actually caused a bug if the user interrupted the wallet selection process. I just pushed a commit that fixes this